src/boot/google-signin.ts

```diff
@@ -0,0 +1,8 @@
+import { boot } from 'quasar/wrappers';
+import GoogleSignInPlugin from "vue3-google-signin"
+
+export default boot(({ app }) => {
+  app.use(GoogleSignInPlugin, {
+    clientId: '680414393195-ikkj5vafetq5q67t499k60n7b1hpiuaf.apps.googleusercontent.com',
```

Comment:
nie lepiej clientId dać do enva?

Reply:
ale to nie jest secret i tak trzeba to podać do frontu i każdy może to podejrzeć.

Reply:
wyniose to do enva